sql/transport.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 var TransportSql = {
-	getCommonBlock: 'SELECT MAX("height") AS "height", "id", "previousBlock", "timestamp" FROM blocks WHERE "id" IN ($1:csv) GROUP BY "id" ORDER BY "height" DESC'
+	getCommonBlock: 'SELECT MAX("height") AS "height", "block_id" AS "id", "previous_block_id" AS "id", "timestamp" FROM blocks WHERE "block_id" IN ($1:csv) GROUP BY "block_id" ORDER BY "height" DESC'
```

Comment:
something wrong here

Reply:
Not fixed, `previous_block_id` should have different alias than `id` which is already taken in that query.